src/ActionsMenuItem.scss

```diff
@@ -1,33 +1,63 @@
+@import 'variables';
+
 .ActionsMenuItem {
-    clear: both;
-    margin: 0 0.5rem;
-    white-space: nowrap;
-}
+    padding: 0.5rem 0.5rem;
+    border-radius: 0.2rem;
+    display: grid;
+    grid-template-columns: min-content 1fr min-content;
+    column-gap: 0.5rem;
+    row-gap: 0;
+    align-items: center;
 
-.ActionsMenuItem__checkbox {
```

Comment:
I realized that clicking right on the checkbox doesn't check/uncheck the checkbox, we need to apply pointer-events: none on .ActionsMenuItem__checkbox. 

```
.ActionsMenuItem__checkbox {
    pointer-events: none; 
}
```

Reply:
Good catch, and fixed.